src/sentry/incidents/endpoints/serializers.py

```diff
@@ -75,15 +75,25 @@ class AlertRuleTriggerActionSerializer(CamelSnakeModelSerializer):
     id = serializers.IntegerField(required=False)
     type = serializers.CharField()
     target_type = serializers.CharField()
+    use_async_lookup = serializers.BooleanField(required=False, default=False)
```

Comment:
It might make more sense to pass this via `context` instead, rather than making it a field. You'll still need to pass it through multiple layers, but it probably fits a little better.

Maybe call it `use_slack_async_lookup` just to be a bit more specific.